src/ExperimentTable.js

```diff
@@ -20,13 +20,19 @@ class ExperimentTable extends React.Component {
       currentPage: 1,
       entriesPerPage: this.entriesPerPageOptions[0],
       selectedSearch: ``,
-      selectedKingdom: ``
+      selectedDropdownFilters: [],
+      experimentTableFilters: this.props.tableFilters.map(filter => {
+        return {
+          label: filter.label,
+          options: _.chain(this.props.aaData).flatMap(filter.dataParam).uniq().value()
+        }
+      })
```

Comment:
Notice that `props` is passed as an argument to `constructor`, so you don’t need to qualify them with `this.`. Replace `this.props` with `props` throughout the constructor.